cmd/cmd.go

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (c) 2022-2023 Dell Inc, or its subsidiaries.
+
+// Package cmd implements the ipsec related CLI commands
+package cmd
+
+import (
+	"log"
+	"os"
+
+	"github.com/opiproject/godpu/cmd/inventory"
+	"github.com/opiproject/godpu/cmd/ipsec"
+	"github.com/opiproject/godpu/cmd/network"
+	"github.com/opiproject/godpu/cmd/storage"
+	"github.com/spf13/cobra"
+)
+
+// AddrCmdLineArg cmdline arg name for address
+const AddrCmdLineArg = "addr"
+
+// TLSFiles cmdline arg name for tls files
+const TLSFiles = "tlsfiles"
+
+// NewCommand handles the cli for evpn, ipsec, invetory and storage
+func NewCommand() *cobra.Command {
+	//
+	// This is the root command for the CLI
+	//
+
+	c := &cobra.Command{
+		Use:   "godpu",
+		Short: "godpu - DPUs and IPUs cli commands",
+		Run: func(cmd *cobra.Command, _ []string) {
+			err := cmd.Help()
+			if err != nil {
+				log.Fatalf("[ERROR] %s", err.Error())
+			}
+			os.Exit(1)
+		},
+	}
+	c.AddCommand(inventory.NewInventoryCommand())
+	c.AddCommand(ipsec.NewIPSecCommand())
+	c.AddCommand(storage.NewStorageCommand())
+	c.AddCommand(network.NewEvpnCommand())
+
+	flags := c.PersistentFlags()
+	flags.String(AddrCmdLineArg, "localhost:50151", "address of OPI gRPC server")
```

Comment:
addr is probably required here, right? Should we mark it with `MarkFlagRequired`?

Reply:
it is not a required flag.

Reply:
true. My oversight